spec.md

```diff
@@ -74,13 +110,44 @@ can be found under the "omero" key in the group-level metadata:
 }
 ```
 
-
 See https://docs.openmicroscopy.org/omero/5.6.1/developers/Web/WebGateway.html#imgdata
 for more information.
 
+### "masks"
+
+The special group "masks" found under an image Zarr contains the key `masks` containing
+the paths to mask objects which can be found underneath the group:
+
+```
+{
+  "masks": [
+    "orphaned/0"
+  ]
+}
+```
+
+Unlisted groups MAY be masks.
+
+### "color"
+
+The `color` key defines an image that is "labeled", i.e. every unique value in the image
+represents a unique, non-overlapping object within the image. The value associated with
+the `color` key is another JSON object in which the key is the pixel value of the image and
+the value is an RGBA color (4 byte, `0-255` per channel) for representing the object:
+
+```
+{
+  "color": {
+    "1": 8388736,
```

Comment:
Do we want to change the image-level friendlier `"color": "0000FF"` and its like, to match?

Reply:
I much prefer the human readability of the hex format. I know what color "0000FF" is but not 8388736!
But I guess it's generally better to align on formats.

Reply:
Me too. Otherwise we could include a CLI option for translating so people know what they mean! :smiley:

Reply:
See https://github.com/ome/omero-cli-zarr/issues/16.